### DISCUSSION TITLE
2 hard drives and Windows 11

### DISCUSSION THREAD

**Discussion by mprattmd2** [sebanc/brunch#1461] [General]
I would like to install a dual boot configuration with Windows 11 on hard drive 1 and brunch/chromeos on hard drive 2

I have searched quite a bit and don't see a best practice for doing this.

Any suggestions?

**Comment by chsc1053** [comment]
I Think this will be a good guide. https://youtu.be/IUuJIkPssEs
At 4:48 in the video while entering your destination, make sure you enter your 2nd hard drive partition
Ex: -dst /mnt/h/chromeos  [where the h drive will be occupying your whole 2nd hard drive]

**Reply by chsc1053** [reply]
Please double check the drive letter correctly. You might erase your windows partition

**Reply by ycavan** [reply]
I recommend booting from your Brunch usb drive and press CTRL+ALT+F2 when you get to the login screen to bring up the console.

login as root ( no password needed ) and type lsblk -e7 to list your devices.

find the empty drive you want to install Brunch onto and type 
#chromeos-install /dev/<drive I want chromeos on >

It will repartition the drive and install chromeos on it.

After that, you'll want to reboot, spam F12 ( or whatever key you use to select the boot drive ) and select your chromeos drive.
